packages/lib-classifier/src/plugins/tasks/dataVisAnnotation/components/DataVisAnnotationTask.js

```diff
@@ -32,7 +32,7 @@ function DataVisAnnotationTask (props) {
 
   return (
     <Box>
-      <StyledText size='small' tag='legend'>
+      <StyledText as='legend' size='small'>
```

Comment:
All these `<legend>` tags have invalid content (#2578.) `<legend>` can only take a single string as content: no headings, paragraphs, lists, links etc.

Reply:
`legend` must be a child of `fieldset` too. These legends aren’t part of fieldsets.